src/components/toast/toast.scss

```diff
@@ -65,7 +66,7 @@
 }
 
 .sbb-toast__overlay-container {
-  position: fixed;
+  position: var(--sbb-toast-container-position, fixed);
```

Comment:
I think the fixed fallback is not needed here, as it is already defined in host

Reply:
I wanted it this way. So I removed it from the host. Is this ok for you?

Reply:
I'd prefer the other way around to be consistent with e.g. `--sbb-toast-container-display` :D